src/cardano-faucet/cardano-faucet.cabal

```diff
@@ -18,7 +18,7 @@ Flag unexpected_thunks
   Default:       False
 
 common base
-  build-depends: base >= 4.14 && < 4.15
+  build-depends: base
```

Comment:
Comment for L4 above (can't add a comment directly there because of [#347](https://github.com/github/roadmap/issues/347)):

> version:                1.35.1

Prior to merge, when we change the flake pin for the next node release that this faucet is compatible with, we probably want to change L4 to the the same version as the flake pin.